database/mml_loader/mml_loader.py

```diff
@@ -170,6 +185,16 @@ def save_geometries(self, geoms: Dict) -> str:
                         )
                         successful_actions += 1
                         break
+            for municipality in municipalities:
+                LOGGER.info(f"Adding geometry to municipality {municipality.value}...")
+                for municipality_id, geom in geoms.items():
+                    if municipality_id == municipality.value:
+                        municipality.geom = geom
+                        LOGGER.info(
+                            f"Geometry added to municipality {municipality.value}"  # noqa
+                        )
+                        successful_actions += 1
+                        break
```

Comment:
For loopin pystyis välttämään, jos hakee suoraan avaimella dictistä:
```suggestion
            for municipality in municipalities:
                LOGGER.info(f"Adding geometry to municipality {municipality.value}...")
                if (geom := geoms.get(municipality.value)):
                    municipality.geom = geom
                    LOGGER.info(
                        f"Geometry added to municipality {municipality.value}"  # noqa
                    )
                    successful_actions += 1
```
Tai sitte nää voisi suoraan päivittää ORM:n `update()`:lla.